### PR TITLE
Israel gov tests data correction

### PR DESCRIPTION
**Pull request by meirroth**
- Added back the source id's of request queries for easier maintenance. (Remove it if you want)
 - NewTestsTaken was showing total new tests including recovery diagnostic tests, fixed it to show new regular virus diagnosis tests taken without recovery tests, screenshot from the source website:
Annotation 2020-08-11 013734

"הנתונים אינם כוללים מידע על בדיקות לאבחון החלמה" 
Translation: The data does not include information on tests for recovery diagnosis.

image

Basically:
`amount` = total new tests including recovery diagnosis. From what can be understood… it's not used on the gov's dashboard.
`amountVirusDiagnosis` = new virus diagnosis tests.
`positiveAmount` = new positive test result.

Thanks

### CODE REVIEW

**Comment by ebwinters** [comment]
>amountVirusDiagnosis = new virus diagnosis tests.
positiveAmount = new positive test result.

What is the difference here though? New virus diagnosis test to me sounds like new positive test. Can you explain the difference, and then we can come up with better names?

**Comment by meirroth** [comment]
> 
> 
> > amountVirusDiagnosis = new virus diagnosis tests.
> > positiveAmount = new positive test result.
> 
> What is the difference here though? New virus diagnosis test to me sounds like new positive test. Can you explain the difference, and then we can come up with better names?

This is from the source API (https://datadashboardapi.health.gov.il/api/queries/_batch)
`amount` = total new tests including recovery diagnosis. From what can be understood… it's not used on the gov's dashboard.
`amountVirusDiagnosis` = new virus diagnosis tests.
`positiveAmount` = new positive test result.

Until now disease.sh /gov/israel data.timeline.newTestsTaken is returning the number from `amount` endpoint that is the amount of virus diagnosis tests taken **including repeated testing of recovered patients.**
Though on the government dashboard and on multiple other Israeli trusted sources like the [HMO](https://www.clalit.co.il/he/your_health/family/Pages/corona_in_israel.aspx) and [news outlets](https://corona.mako.co.il/), show the number of tests without the repeated testing that matches the `amountVirusDiagnosis` endpoint. 

So I want to correct the disease.sh data.timeline.newTestsTaken to reflect the same number which is without the repeated testing.
We can have both the numbers somehow, I just did not know what to name it.
Here is a suggestion:
data.timeline.newTotalTestsTaken -> `amount`
data.timeline.newVirusTestsTaken -> `amountVirusDiagnosis`

Thoughts?

BTW any updates with [If no data is returned, serve stale data instead of an error](https://github.com/disease-sh/API/commit/ae1ebb2d2ecc6c481e966a00ac7dcffcaf56cd66#diff-991a44f4dbe805f7e115ab3c18ae1f8b)

Thanks Ethan
